decentraland/App.js

```diff
@@ -44,6 +46,11 @@ Is the 'config/default.js' file correct?`;
     }
   }
 
+  async _loadAccount() {
+    const account = await recoverAccount();
+    if (account != null) store.dispatch(setAccount(account));
```

Comment:
Hmmm, are you suggesting that we can always get the account from the storage? If yes, I think that will work but I'm not sure if it will be a good practice since the recommendation is having a "single source of truth".

Reply:
Yeah, I was. But I think your argument is a convincing one, so I'll resolve this.